shared/openId4VCI/Utils.ts

```diff
@@ -208,11 +215,18 @@ export const getCredentialIssuersWellKnownConfig = async (
     } else if (response?.credentials_supported[0].order) {
       fields = response?.credentials_supported[0].order;
     } else {
-      fields = Object.keys(
-        response?.credentials_supported[0].credential_definition
-          .credentialSubject,
+      const supportedCredentialTypes = credentialTypes.filter(
```

Comment:
It can be simply mentioned as credentialTypes[1] as second value would be proper credential type and first type would be "VerifiableCredential".